shared/api/certificate.go

```diff
@@ -26,6 +26,12 @@ type CertificatesPost struct {
 	// Server trust password (used to add an untrusted client)
 	// Example: blah
 	Password string `json:"password" yaml:"password"`
+
+	// Whether to use token
```

Comment:
That's incorrect, it doesn't intake that we're consuming a token, it indicates that we're issuing a token.

So `Whether to create a join token` would be better